### PR TITLE
tests failed because failing to create raw cert data from the signature

### PR DESCRIPTION
**Pull request by weijjia**
The signature validation tests need to read all the raw certs. The tests only make sure the signature hash matches, which cause the test failings when creating X509 certs.

### CODE REVIEW

File: adal/src/androidTest/java/com/microsoft/aad/adal/Util.java [hunk 1]
```diff
@@ -39,6 +39,8 @@
...
...6jO5vgDVV9OqRLLcAwrlewqua9oTDbAp/mDldztp//Z+8XiY6j/AJCKFvn+cA4s6s5kYj/jsK4/wt9nfo5aD9vRzE2j2IIH1T0Qj6NLTNxB7+Ij6dykE8QHJ7Vd/Y5af9QZwXyyPdSvwqhvKafS0baSqy1gLaNLA/gc/1Sh/ASXaDEhKHHAsLChkVFCE7cPwKPnBHudNBmS6HQ6Zo3UMwYVQVe7u+6jjvfo4gqmZglMhhzhauekNrHV91E+GkY3NGH2cHDEbpbl0JAAdWsI4jtJSN8c9Y8lSX00D7KdQ2NJhYl7mJsS10/3Ex1HYr8nDRq/IlAhGdSVC/qc9RktfYiYcmfZ/Iel5n+KkQt1svrF1TDCHYg/bcC7BhCwlaoa4Nu0hvLHvSbrsnB+gKtovCCilswPwCnDdAYmSMnwsAtBwJXqxD6HXbBCNX4A+qUrR+sYhmFa8jIVzAXa4I3iTvVQkTvrf9YriP7AgMBAAEwDQYJKoZIhvcNAQELBQADggIBAEdMG13+y2OvUHP1lHP22CNXk5e2lVgKZaEEWdxqGFZPuNsXsrHjAMOM4ocmyPUYAlscZsSRcMffMlBqbTyXIDfjkICwuJ+QdD7ySEKuLA1iWFMpwa30PSeZP4H0AlF9RkFhl/J9a9Le+5LdQihicHaTD2DEqCAQvR2RhonBr4vOV2bDnVParhaAEIMzwg2btj4nz8R/S0Nnx1O0YEHoXzbDRYHfL9ZfERp+9I8rtvWhRQRdhh9JNUbSPS6ygFZO67VECfxCOZ1MzPY9YEEdCcpPt5rgMEKVh7VPH14zsBuky2Opf6rGGS1m1Q26edG9dPtnAYax5AIkUG6cI3tW957qmUVSnIvlMzt6+OMYSKf5R5fdPdRlH1l8hak9vMxO2l344HyD0vAmbk01dw44PhIfuoq2qNAIt3lweEhZna8m5s9r1NEaRTf1BrVHXloAM+sipd5vQNs6oezSCicU7vwvUH1hIz0FOiCsLPTyxlfHk3ESS5QsivJS82TLSIb9HLX07OyENRRm8cVZdDbz6rRR+UWn1ZNEM9q56IZ+nCIOCbTjYlw1oZFowJDCL1IH8i7nhKVGBWf7TfukucDzh8ThOgMyyv6rIPutnssxQqQ7ed6iivc1y4Graihrr9n2HODRo3iUCXi+G4kfdmMwp2iwJz+Kjhyuqf7lhdOld6cs";
```
Comment (addev-ashish) [inline adal/src/androidTest/java/com/microsoft/aad/adal/Util.java]: uppercase

Comment (weijjia) [inline adal/src/androidTest/java/com/microsoft/aad/adal/Util.java]: ah, my bad. Copied from my local variable.

File: adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java [hunk 2]
```diff
@@ -675,9 +675,7 @@ private PackageManager getMockedPackageManager() throws PackageManager.NameNotFo
     }
 
     private void prepareAuthForBrokerCall() throws NoSuchAlgorithmException {
```
Comment (addev-ashish) [inline adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java]: can this function now throw NoSuchAlgorithmException

Comment (weijjia) [inline adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java]: no. good catch, will remove.

**Comment by addev-ashish** [comment]
:shipit:

**Pull request merged**